q2_moshpit/busco/fetch_busco_db.py

```diff
@@ -0,0 +1,38 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2023, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+from q2_moshpit._utils import colorify, run_command
+from q2_moshpit.busco.types import BuscoDatabaseDirFmt
+
+
+def fetch_busco_db(virus: bool, prok: bool, euk: bool) -> BuscoDatabaseDirFmt:
+    busco_db = BuscoDatabaseDirFmt(path=None, mode='w')
+
+    # Parse kwargs
+    if all([virus, prok, euk]):
+        args = ["all"]
+    else:
+        args = []
+        variable_and_flag = [
+            ('virus', virus), ('prokaryota', prok), ('eukaryota', euk)
+        ]
+        for variable_name, flag in variable_and_flag:
+            if flag:
+                args.append(variable_name)
```

Comment:
Maybe if you do this minor change it will be more concise and readable by leveraging a list comprehension. But its really up to you, just a suggestion. 
```suggestion
    else:
        variable_and_flag = [
            ('virus', virus), 
            ('prokaryota', prok), 
            ('eukaryota', euk)
        ]
        args = [name for name, flag in variable_and_flag if flag]
```

Reply:
I like it!